web/src/components/vis/PcaPage/PcaPage.vue

```diff
@@ -0,0 +1,158 @@
+<script>
+import ScorePlot from './ScorePlot.vue';
+import ScreePlot from './ScreePlot.vue';
+import LoadingsPlot from './LoadingsPlot.vue';
+import VisTileLarge from '@/components/vis/VisTileLarge.vue';
+import LayoutGrid from '@/components/LayoutGrid.vue';
+
+export default {
+  components: {
+    ScorePlot,
+    ScreePlot,
+    LoadingsPlot,
+    VisTileLarge,
+    LayoutGrid,
+  },
+
+  props: {
+    id: {
+      type: String,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      pcXval: '1',
+      pcYval: '2',
+      numComponentsVal: '10',
+      showEllipses: true,
+      showCrosshairs: true,
+      showCutoffs: true,
+      showScree: true,
+      showScore: true,
+      showLoadings: true,
+    };
+  },
+
+  computed: {
+    pcX() {
+      return Number.parseInt(this.pcXval, 10);
+    },
+
+    pcY() {
+      return Number.parseInt(this.pcYval, 10);
+    },
+
+    numComponents() {
+      return Number.parseInt(this.numComponentsVal, 10);
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+vis-tile-large(title="Principal Component Analysis", :loading="false", expanded)
+  template(#controls)
+    v-toolbar.darken-3(color="primary", dark, flat, dense, :card="false")
+      v-toolbar-title PC selector
+    v-card.mb-3.mx-3(flat)
+      v-card-actions
+        v-layout(column)
+          v-text-field.py-2(
+              hide-details,
+              type="number",
+              label="PC (X Axis)",
+              min="1",
+              outline,
+              :disabled="!showScore && !showLoadings",
+              v-model="pcXval")
+          v-text-field.py-2(
+              hide-details,
+              type="number",
+              label="PC (Y Axis)",
+              min="1",
+              outline,
+              :disabled="!showScore && !showLoadings",
+              v-model="pcYval")
+
+    v-toolbar.darken-3(color="primary", dark, flat, dense, :card="false")
+      v-toolbar-title.switch-title Scree Plot
+        v-switch.switch(v-model="showScree", color="white", hide-details)
+    v-card.mb-3.mx-3(flat)
+      v-card-actions
+        v-layout(column)
+          v-text-field.py-2(
+            :disabled="!showScree",
+            hide-details,
+            type="number",
+            label="Number of PCs",
+            min="1",
+            outline,
+            v-model="numComponentsVal")
```

Comment:
missing indentation of two space to be valid

Reply:
c9cafed should fix this and the other lint problem.